docs/source/caching/declarative-ids.mdx

```diff
@@ -1,100 +1,95 @@
 ---
-title: Declarative cache ids
+title: Declarative cache IDs in Apollo Android
```

Comment:
Question(s): Let's say I set up a `fieldPolicy` to get an object's ID, but that object isn't in the cache. So now I hit the network and the object is returned. When assigning the cache ID for that object, is the `fieldPolicy` value used, or is the `typePolicy` value used (assuming both are defined)? What if there's a `fieldPolicy` for a field but _not_ a `typePolicy` for the type returned by that field?

Reply:
`@typePolicy` is the one used after the network. `@fieldPolicy` doesn't have access to the object because it is used before the network.

Usually, you would setup a `@typePolicy` always and optionally a `fieldPolicy`. I don't think there's a valid use case where `@fieldPolicy` is defined but not `@typePolicy`